src/main/java/com/google/sps/data/Coordinates.java

```diff
@@ -27,6 +27,11 @@ public Coordinates() {
     this.lat = -1;
   }
 
+  public void setCoordinates(double lng, double lat) {
```

Comment:
Since this is a PDS, and the lat/lng are already public, why do you need a setter? Just directly:
```
Coordinates myCoord = new Coordinates();
myCoord.lat = lat;
myCoord.lng = lng;
```